packages/connectivity_plus/connectivity_plus/darwin/Classes/ConnectivityPlugin.m

```diff
@@ -8,8 +8,8 @@
 #import "connectivity_plus-Swift.h"
 #endif
 
-@implementation ConnectivityPlusPlugin
+@implementation ConnectivityPlugin
 + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
-  [SwiftConnectivityPlusPlugin registerWithRegistrar:registrar];
+  [SwiftConnectivityPlugin registerWithRegistrar:registrar];
 }
```

Comment:
I think that this can cause troubles with people using the original connectivity plugin, if I remember correctly. I'd suggest not making this refactor now, and instead keeping the original file name.

Reply:
Yes, naming has to be reverted. Don't get why files were renamed here in the first place. Plugins are called `Plus Plugins`, so, please, don't touch naming for things like these ones.

Reply:
Agreed. Reverted the name to the original one. Since we've merged ios & macos codebases into one then the macos plugin was renamed from `ConnectivityPlugn` to `ConnectivityPlusPlugin`. I hope it going to be ok.